.env.example

```diff
@@ -7,25 +7,3 @@ APP_KEY=
 APP_DEBUG=true
 APP_LOG_LEVEL=debug
 APP_URL=http://localhost
-
```

Comment:
If Twilio credentials are needed, you should include them in the .env.example file too.